packages/common/src/content/hostedServiceUtils.ts

```diff
@@ -86,10 +86,26 @@ export function isAGOFeatureServiceUrl(url: string): boolean {
   // TODO: we should really centralize this regex somewhere
   const FEATURE_SERVICE_URL_REGEX = /(feature)server(\/|\/(\d+))?$/i;
   return (
-    !!url && url.includes("arcgis.com") && FEATURE_SERVICE_URL_REGEX.test(url)
+    !!url &&
+    url.includes("arcgis.com") &&
+    FEATURE_SERVICE_URL_REGEX.test(url) &&
+    !isSecureProxyServiceUrl(url)
   );
 }
 
+/**
+ * Portal secure proxy services are identified by looking for these patterns in the `url`:
```

Comment:
Let's add to the comment so future devs know where this logic came from, something like:
`This function was given to us by the JSAPI team, so we trust the regex's are correct`